magprime/__init__.py

```diff
@@ -1,67 +1,41 @@
 from uber.common import *
 from magprime._version import __version__
 
-config = parse_config(__file__)
-mount_site_sections(config['module_root'])
-static_overrides(join(config['module_root'], 'static'))
-template_overrides(join(config['module_root'], 'templates'))
-
-
-
```

Comment:
The automated emails have been moved into their own ``magprime.automated_emails`` module.